app/controllers/api/functions.php

```diff
@@ -650,6 +767,38 @@
             throw new Exception('Tag not found', 404);
         }
 
+        // Request executor to delete tag containers
```

Comment:
Should be moved to a worker, no need to wait for response. We might have to introduce a `deleting` status to prevent conflicts while being deleted.